admin/formatter/class-metabox-formatter.php

```diff
@@ -148,14 +151,28 @@ private function get_defaults() {
 	}
 
 	/**
-	 * Returns a link to the settings page, if the user has the right capabilities.
+	 * Returns a link to the General Settings page, if the user has the right capabilities.
 	 * Returns an empty string otherwise.
 	 *
-	 * @return string The settings link.
+	 * @return string The General Settings link.
 	 */
 	private function get_settings_link() {
 		if ( current_user_can( 'manage_options' ) ) {
-			return admin_url( 'options-general.php' );
+			return esc_url( admin_url( 'options-general.php' ) );
+		}
+
+		return '';
+	}
+
+	/**
+	 * Returns a link to the Reading Settings page, if the user has the right capabilities.
+	 * Returns an empty string otherwise.
+	 *
+	 * @return string The Reading Settings link.
+	 */
+	private function get_reading_settings_link() {
```

Comment:
Same for this method: can be removed, as it was only used for the `readingSettingsLink` used in the Gutenberg notice.